source/sass_api_reference/index.html.md.erb

```diff
@@ -0,0 +1,65 @@
+---
+title: Sass API Reference
+weight: 65
+---
+
+# Sass API Reference
+
+<% format_sassdoc_data(data).each do |heading, group| %>
+
+## <%= group_heading(heading) %>
+
+<% group.each do |doc| %>
+
+### <%= doc_heading(doc) %>
+<span class="govuk-secondary-text">(<%= doc.context.type %>)</span>
+
+<%= doc.description %>
+
+<% if doc.parameter %>
+
+#### Parameters
+
+| Name | Description | Type | Default value |
+| ---- | ----------- | ---- | ------------- |
+<% parameters_table(doc.parameter).each do |param| %>
+|  <%= param.name %> | <%= param.description %> | <%= param.type %> | <%= param.default_value %> |
+<% end %>
+<% end %>
+
+<% if doc.deprecated || doc.context.type == 'mixin' || doc.example %>
```

Comment:
This is a little strange as it can mean you have a h4 usage heading followed immediately by a h4 examples heading, if you have examples but it's not a mixin or deprecated.

https://deploy-preview-15--govuk-frontend-docs-prototype.netlify.com/sass_api_reference/#govuk-spacing

Reply:
agreed will fix